med_io/generate_label.py

```diff
@@ -47,7 +48,7 @@ def make_reg_label(label, patch_pos, patch_size):
         reg_label_=[]
         threshold_=[]
 
-        for idx in range(config['body_identification_n_classes'] - 1):
+        for idx in range(config['num_classes'] - 1):
```

Comment:
why rebranding needed? `num_classes` not specified in config

Reply:
I did not know that the variable "body_identification_n_classes" was the number of classes for my dataset. I used "num_classes" because MELANOM dataset has different labels, but this can be adapted, or num_classes can be added to config? I can still change this part

Reply:
okay, this then actually reflected in `len(config['name_output_channel'][config['dataset']])`
I changed it